src/ansys/mapdl/core/plotting/theme.py

```diff
@@ -0,0 +1,155 @@
+# Copyright (C) 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+"""Store parameters for a PyMAPDL-specific theme for pyvista"""
+
+import numpy as np
+
+try:
+    from matplotlib.colors import ListedColormap
+
+    _HAS_MATPLOTLIB = True
+except ModuleNotFoundError:
+    _HAS_MATPLOTLIB = False
+
+
+from ansys.mapdl.core import _HAS_PYVISTA
+
+if _HAS_PYVISTA:
+    from pyvista.plotting.colors import get_cycler
+
+    try:
+        from pyvista.plotting.themes import Theme
+
+    except ImportError:
+        from pyvista import __version__ as pyvista_version
+
+        if "0.40" in pyvista_version:
+            from pyvista.themes import Theme
+        else:  # older versions
+            from pyvista.themes import DefaultTheme as Theme
+
+    base_class = Theme
+
+else:  # pragma: no cover
+
+    class myEmptyClass:
+        pass
+
+    base_class = myEmptyClass
+
+
+MAPDL_colorbar = (
+    np.array(
+        [
+            [0, 0, 255],
+            [0, 178, 255],
+            [0, 255, 255],
+            [0, 255, 178],
+            [0, 255, 0],
+            [178, 255, 0],
+            [255, 255, 0],
+            [255, 178, 0],
+            [255, 0, 0],
+        ],
+        dtype=float,
+    )
+    / 255
+)
+
+if _HAS_MATPLOTLIB:
+    PyMAPDL_cmap: ListedColormap = ListedColormap(MAPDL_colorbar, name="PyMAPDL")
+
+
+def get_ansys_colors(N=9):
+    if not _HAS_MATPLOTLIB:
+        raise ModuleNotFoundError(
+            "'matplotlib' package is needed for 'get_ansys_colors'."
+        )
+    return np.array([PyMAPDL_cmap(i) for i in range(N)])
+
+
+class MapdlTheme(base_class):
+    """PyMAPDL-specific theme for pyvista.
```

Comment:
```suggestion
    """Provides the PyMAPDL-specific theme for PyVista.
```
Class descriptions should start with a verb ending in an "s" or "es".